# HG changeset patch
# User Rik <rik@octave.org>
# Date 1347819461 25200
#      Sun Sep 16 11:17:41 2012 -0700
# Node ID 2b72343ed9c43a5047632ed294d856bc881f8736
# Parent  b99c52303d0bf22ca253adff218830d939973c4e
hgignore: Update patterns to reflect directory changes to source tree.

* hgignore: Update patterns to reflect directory changes to source tree.

diff --git a/.hgignore b/.hgignore
--- a/.hgignore
+++ b/.hgignore
@@ -1,44 +1,43 @@
 syntax: regexp
 # The recurrent (^|/) idiom in the regexps below should be understood
 # to mean "at any directory" while the ^ idiom means "from the
 # project's top-level directory".
 
 # gnulib makes these silly backup files
 ^.*~
 
-# This one gets created by gnulib during the build, contains no Octave
-# files.
+# This directory gets created by gnulib during the build. 
+# It contains no Octave files.
 ^gnulib/
 
 # These directories mostly contain cruft during build time, but they
 # do contain some Octave code, so we gotta be a bit more careful about
 # what we ignore here
 ^libgnu/.*\.([hc]$|sin$|valgrind$|charset$)
 ^m4/(?!ax_).+\.m4$
 
 # Emacs tools create these
 (^|/)TAGS$
 (^|/)semantic.cache$
 
 (/|^)Makefile\.in$
 ^INSTALL$
 
-^aclocal\.m4
+^aclocal\.m4$
 ^build-.*($|/)
 ^configure$
 ^autom4te\.cache($|/)
 ^config\.h\.in$
 
 # e.g. doc/faq/OctaveFAQ.info
 #      doc/interpreter/octave.info-4
 ^doc/.*\.info(-\d)?$
 
 ^doc/\w*/stamp-vti$
 ^doc/\w*/version\.texi$
 ^doc/interpreter/images\.mk$
 
-# e.g. liboctave/smx-op-inc.mk
+# e.g. liboctave/operators/smx-op-inc.mk
 ^liboctave/operators/\w+-op-\w+\.mk$
-^liboctave/\w+-op-\w+\.mk$
 
 ^libinterp/dldfcn/module\.mk$
